data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -93,13 +93,21 @@ then
 
 	rm --recursive --force api-bootstrap
 
+  # A more desireable end state would be require wiring up the infrastructure in some way and read it from rendered-manifest-dir
```

Comment:
This is available in [`InfrastructureStatus.InfrastructureTopology`](https://github.com/openshift/api/blob/59fa376de7cb668ddb95a7ee4e9879d7f6ca2767/config/v1/types_infrastructure.go#L107). Do you prefer the current implementation for now?

Reply:
> This is available in [`InfrastructureStatus.InfrastructureTopology`](https://github.com/openshift/api/blob/59fa376de7cb668ddb95a7ee4e9879d7f6ca2767/config/v1/types_infrastructure.go#L107). Do you prefer the current implementation for now?

Yeah, given scope of change, this is our best 4.15 solution.